### PR TITLE
Gitpod integration to Venom

### PR DESCRIPTION
**Pull request by scraly**
In order to facilitate the possibility of contributing to Venom, without having to configure the setup on your computer, with everything ready to use, this PR adds integration to GitPod.

GitPod is an Open Source IDEaaS, surprisingly simple and effective solution The principle? A cloud-based development environment, based on Git, Eclipse Theia and Docker, allowing you to develop, test and deploy your code right from a browser window.

If you want to know more about Gitpod:
https://www.youtube.com/watch?v=pDu_rTh_s-0

With this feature, contributors will no longer need to:
- install Go in their computer
- use a Go correct version or update to a good one
- know the command to start the project

To contribute, they will only need to click on "Open to GitPod"... and start to contribute :).

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-venom-a#811.0 ✘  
* Build
  * Build ✔ 
* Tests
  * Acceptance Tests ✘

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+tasks:
+  - init: go get -v -t -d ./...
```
Comment (yesnault) [inline .gitpod.yml:2]: Is it possible to get the latest release ?
I'm not a fan of launching venom by bypassing the entire CI. Is it possible to run something like that:

```
LAST_RELEASE=$(curl -s https://api.github.com/repos/ovh/venom/releases | grep tag_name | head -n 1 | cut -d '"' -f 4) && \
DL=$(curl -s https://api.github.com/repos/ovh/venom/releases | grep ${LAST_RELEASE} | grep browser_download_url | grep -E 'linux-amd64' | cut -d '"' -f 4) &&
curl -L $DL -o venom
```

**Review by yesnault — CHANGES_REQUESTED** [review]

File: .gitpod.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    command: go run cmd/venom/main.go -h
```
Comment (fsamin) [inline .gitpod.yml:3]: If the goal of this integration is to have dev environment for dev, i suggest to try to use the Makefile to build and maybe run the tests ?

Comment (scraly) [reply .gitpod.yml:3]: The goal is to have an IDE in the cloud/in the browser. But not only an IDE but a complete dev environment.
No need to install, in your local machine(s), the correct version of Go, all the tools, the extensions in VSCode... it's one of the power of GitPod.

Comment (fsamin) [reply .gitpod.yml:3]: Great ! 
So I guess we should try to only relies on the Makefile
We don't need the `go get` and `make build` should be fine

Comment (scraly) [reply .gitpod.yml:3]: When I am developing an application in Go, I don't want to build my binary but to make a go run in order to test in live my changes, for example.
I just want to have my go dependencies in order to then execute `go run` command.

**Comment by fsamin** [comment]
I am closing this because it has been merged in #559

**Pull request closed without merging**